scripts/syspoll

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import json
+import time
+
+progname = sys.argv[0]
+
+outfile = '/var/platform/system'
+tmpfile = '/var/platform/system.tmp'
+
+ticks_per_sec = os.sysconf(os.sysconf_names['SC_CLK_TCK'])
+
+while True:
```

Comment:
I see it as a pretty handy tool to collect all info in human readable format. 

Can you help us understand the use case of keeping a file in sync with system state, by running every second ?

[Sorry for the delay. The comments from other PR drowned the rest :-) ]

Reply:
This was the initial implementation of a back-end for a CLI command to show system status -- the front-end would simply read a known file, which is constantly updated by this back-end daemon.
I would say that, one day soon, the redis DB should be used, and be updated by the daemon, rather than a file.